benchmarks/BunnymarkV1Sprites/d/BunnymarkV1Sprites.d

```diff
@@ -8,11 +8,11 @@ import godot, godot.core;
 import godot.texture, godot.node2d, godot.sprite;
 import godot.resourceloader;
 
-class BunnymarkV1Sprites : GodotScript!Node2D 
+class BunnymarkV1Sprites : GodotScript!Node2D
 {
     alias owner this;
 
-    @OnReady!((){return cast(Texture)ResourceLoader.load("res://images/godot_bunny.png");})
+    @OnReady!((){return ResourceLoader.load("res://images/godot_bunny.png").as!Texture;})
```

Comment:
`load` returns a `Ref`, so assigning it to a `Texture` variable makes the refcount go to 0. It'll work if you change the variable to `Ref!Texture` and the cast to `as!(Ref!Texture)`.

I need to make the unsafe implicit conversion a compile-time error in the future to prevent this kind of bug.

Reply:
Thanks! Shouldn't Godot API methods then also accept `Ref!Texture` and not `Texture`? How do I pass the texture to `drawTexture` method correctly?

Reply:
Refs should still be passed directly to those methods. I don't know yet how much the bindings' implementation will change, but this will always be valid:  
```d
Ref!Texture texture;
owner.drawTexture(texture, Vector2(0f, 0f));
```